config.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  port:
+    description: |
+      Prometheus-edge-hub port
+    type: int
+    default: 9091
+  grpc-port:
+    description: |
+      Prometheus-edge-hub grpc-port
+    type: int
+    default: 0
+  limit:
```

Comment:
This config name is imo too generic: How about `max-metrics-count`?

Reply:
It's true that it's generic but I'm not sure about using a different name than what's documented in the actual service. If somebody looks up prometheus-edge-hub, they'll see the config `limit`, not `max-metrics-count` or anything else.

Reply:
How about `metrics-count-limit`? And then in the description state that it is the same as `limit`

Reply:
Also per the style guide which'll be shortly published, I'd prefer "metrics_count_limit" :bike: :house:

Reply:
Agreed, fixed